README.md

```diff
@@ -1,2 +1,54 @@
 # action-baseline
-A GitHub Action for running the OWASP ZAP Baseline scan
+A GitHub Action for running the OWASP ZAP Baseline scan. 
+
+The ZAP baseline scan action scans a target URL for vulnerabilities and create issue in the relevant Github Repository
+with the link to the latest report. The generate reports will be committed to repository.
+
+Checkout the `action.yml` file for additional parameters on how to use it.
```

Comment:
Better to describe them here? As suggested in: https://help.github.com/en/actions/building-actions/creating-a-javascript-action#create-a-readme